charts/seed-monitoring/charts/core/charts/prometheus/rules/node-exporter.rules.yaml

```diff
@@ -65,6 +65,19 @@ groups:
       description: Root filesystem device on instance {{ $labels.instance }} is almost full.
       summary: Node's root filesystem is almost full
 
+  # alert if the /run filesystem is almost out of inodes
+  - alert: NodeFilesystemAlmostOutOfFiles
+    expr: node_filesystem_files_free{mountpoint="/run"} / node_filesystem_files{mountpoint="/run"} * 100 < 5 and node_filesystem_readonly{mountpoint="/run"} == 0
```

Comment:
Just for the record: the `node_filesystem_files_free` metric is added in https://github.com/gardener/gardener/pull/6424.

Reply:
let's keep this one on hold til 6424 is merged.